yaksh/views.py

```diff
@@ -2808,3 +2816,27 @@ def get_user_data(request, course_id, student_id):
         context = Context(data)
         data = template.render(context)
     return HttpResponse(json.dumps({"user_data": data}), **response_kwargs)
+
+
+@login_required
+@email_verified
+def download_course(request, course_id):
```

Comment:
Should be offline_view_course or download_offline_course

Reply:
I feel this is ok because the same function can be used to download course with YAML content with another argument as a type which can either be `html` or `yaml`. In that case suggested function name will not be appropriate.